Packs/CortexXDR/ReleaseNotes/5_2_1.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Added the **Remove legacy incident fields** integration parameter. You will need to enable this parameter in your instance settings.
```

Comment:
```suggestion
- Added the **Remove legacy incident fields** integration parameter. You will need to enable this parameter in your instance settings in case you want to remove old legacy incident fields.
```
Is there any way to explain which fields will be removed the snake case vs the camel case?

Reply:
updated the rn